requirements.txt

```diff
@@ -12,13 +11,20 @@ urllib3==1.26.0
 pydantic==1.10.11
 httpx==0.25.0
 semver==3.0.2
-scipy==1.10.1
 memory-profiler==0.61.0
-cachetools==5.3.1
-pynvml==11.5.0 # For cuda utilization
-readerwriterlock==1.0.9
+pynvml==11.5.0
 kazoo==2.10.0
 pycurl==7.45.3
 certifi==2019.11.28
 transformers==4.41.2
-optimum==1.20.0
\ No newline at end of file
+optimum==1.20.0
+
+# LanguageBind
+
+eva-decord==0.6.1
+einops==0.6.1
+pytorchvideo==0.1.5
+torchaudio==0.12.1
+SoundFile==0.12.1
+python-magic==0.4.27
+ffmpeg-python==0.2.0
```

Comment:
Can be removed once the base image has these deps

Reply:
done